doc/server/tutorial.rst

```diff
@@ -221,8 +221,13 @@ Aggregate sensors
 To provide a sensor which has its reading derived from that of a set of other
 sensors, such as a total, average or general "device status" sensor, subclass
 :class:`.AggregateSensor`, and implement
-:func:`.~AggregateSensor.update_aggregate` to compute the sensor reading from
+:func:`.!update_aggregate` to compute the sensor reading from
 changes in sensor values.
+In order to avoid circular references, a :func:`!filter_aggregate` method is
```

Comment:
Again, it should be `:meth:` rather than `:func:`, and if you use `~.AggregateSensor.filter_aggregate` it should create the link.